sdk/src/test/java/io/dapr/it/services/HelloWorldGrpcStateService.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it.services;
+
+import com.google.protobuf.Any;
+import com.google.protobuf.ByteString;
+import io.dapr.DaprGrpc;
+import io.dapr.DaprGrpc.DaprBlockingStub;
+import io.dapr.DaprProtos.SaveStateEnvelope;
+import io.dapr.DaprProtos.StateRequest;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+/**
+ * Simple example, to run:
+ * mvn clean install
+ * dapr run --grpc-port 50001 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.Example
+ */
+public class HelloWorldGrpcStateService {
+
+    public static void main(String[] args) {
+        ManagedChannel channel = ManagedChannelBuilder.forAddress("localhost", 50002).usePlaintext().build();
```

Comment:
DAPR_HTTP_PORT or DAPR_GRPC_PORT env vars.

Reply:
I already change this code, with auto-generate free ports code